package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
In one of your PRs, I recommend updating this to 2.x.x (maybe 2.7.2 to retain some connection with past work?).

Why v2? Because v1 has already been live for a while, and you're adding a new filing type.

Open to other ideas.

Reply:
I bumped it up to `0.7.x` with this new filing type, but i agree. 
Perhaps when we release Change Firm filings, we can then set the version to `2.0.0`.

Reply:
When to change a version number is a good question...

In my opinion/experience, we should change it when we start a new feature. Let's say we go with 2.0.0 here. As the feature progresses, we will go to 2.0.1 or maybe 2.1.0. Eventually the production-ready version of the feature might be, say, 2.1.10. Then we repeat for future features. But what if we need to back-patch a bug (using a bugfix branch)? Then it makes sense that the fixed version for that feature is 2.1.11. This allows the "final version" of a feature to always be 2.something.

IMO, it makes less sense to say "2.0.0 is the production-ready version for this feature". First, what about changes and fixes? Second, the next feature will be in 2.x.x instead of its own version.

Again, my opinion, open to other ideas.

Reply:
Ok, i'm pretty impartial.
Updated to 2.0.0!